docs/gallery/plot_convert_nwb_hdf5.py

```diff
@@ -107,9 +105,10 @@
 #
 # Using the same approach as above, we can now convert our Zarr file back to HDF5.
 
-with NWBZarrIO(zarr_filename, mode='r') as read_io:  # Create Zarr IO object for read
-    with NWBHDF5IO(hdf_filename, 'w') as export_io:  # Create HDF5 IO object for write
-        export_io.export(src_io=read_io, write_args=dict(link_data=False))  # Export from Zarr to HDF5
+with suppress(Exception):  # TODO: This is a temporary ignore on the convert_dtype exception.
```

Comment:
@mavaylon1 can you please create an issue for this so we don't forget to check on the dtype conversion issue (and include links to the lines where the issue is being suppressed)

Reply:
Done